lib/go-tc/crstates.go

```diff
@@ -44,6 +44,7 @@ type IsAvailable struct {
 	DirectlyPolled bool      `json:"-"`
 	Status         string    `json:"status"`
 	LastPoll       time.Time `json:"lastPoll"`
+	LastPollv6     time.Time `json:"lastPollv6"`
```

Comment:
It's true that `v` is not capitalized in `IPv6`, but variables are capitalized according to CamelCase, so `V` should be capitalized in both Go and JSON fields.